### PR TITLE
docs - c7n-kates helm deployment docs

### PR DESCRIPTION
**Pull request by djeebus**
I also added `--host` to the list of options taken by c7n-kube, which defaults to listening to localhost, for standard practice security purposes.

### CODE REVIEW

File: tools/c7n_kube/readme.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Custodian Kubernetes Support
 
+Cloud custodian can run policies directly inside your cluster, reporting on 
+resources that violate those policies, or blocking them altogether.
 
-Work in Progress - Not Ready For Use.
+# Running the server 
 
+c7n-kube can be run and installed via poetry. `poetry install && poetry run c7n-kates`.
```
Comment (thisisshi) [inline tools/c7n_kube/readme.md:8]:
```suggestion
c7n-kates can be run and installed via poetry. `poetry install && poetry run c7n-kates`.  
```

File: tools/c7n_kube/readme.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
 # Custodian Kubernetes Support
 
+Cloud custodian can run policies directly inside your cluster, reporting on
```
Comment (thisisshi) [inline tools/c7n_kube/readme.md:3]:
```suggestion
Cloud Custodian can run policies directly inside your cluster, reporting on 
```

**Review by thisisshi — CHANGES_REQUESTED** [review]

**Comment by djeebus** [comment]
@thisisshi oops, thanks for the :eye: !

**Review by kapilt — COMMENTED** [review]
lgtm overall, but needs fixing re failing tests.

**Review by kapilt — APPROVED** [review]
lgtm

**Pull request merged**